Exadel.Compreface/Services/FaceVerificationService.cs

```diff
@@ -28,7 +29,9 @@ public async Task<FaceVerificationResponse> VerifyImageAsync(FaceVerificationReq
                 face_plugins = string.Join(",", request.FacePlugins),
                 status = request.Status,
             });
-        
+
+        requestUrlWithQueryParameters.Port = Convert.ToInt32(_comprefaceConfiguration.Port);
```

Comment:
I think instead of converting request url port here you better just declare requestUrl in the following manner:
```
var requestUrl = $"{_comprefaceConfiguration.Domain}:{_comprefaceConfiguration.Port}/api/v1/verification/verify"
```